common/maliit/namespace.h

```diff
@@ -44,6 +44,9 @@ namespace Maliit {
         //! only integer numbers allowed
         NumberContentType,
 
+        //! only numbers and formatted characters
+        FormattedNumberContentType,
```

Comment:
Pondering if we should just use Qt::inputMethodHint which offers more types and hint combinations.

On the history side not entirely sure did we even add this before Qt got that hint of its own.

Reply:
I'm not sure how much sense it makes, because the Wayland protocols are so unstable right now, and Phosh devs keep trying to push their own changes which break the existing protocols, and Qt::InputMethodHints doesn't have flags for everything, so I can only see it getting more complicated to keep updated, along with I don't want to have to block on getting things into Qt, maybe backported, etc… if we want to add anything else in the future.

Reply:
Could alternatively be a similar enum. But then again Qt::InputMethodHints should not block adding new extensions, those can be presented with other properties. We already do pass the content type here but also the input method hints.

Actually looks like I've halfway marked this as deprecated in 2013 with the Qt5 port. minputcontext.cpp / MInputContext::getStateInformation()

Also curious on what those wayland changes would be. The input hint side has seemed relatively stable for many years.